src/Presentation/AmountFormatter.php

```diff
@@ -14,8 +14,8 @@ class AmountFormatter {
 	private $locale;
 
 	private $localeData = [
-		'de_DE' => [2, ',', '' ],
-		'en_US' => [2, '.', '' ]
+		'de_DE' => [2, ',', ''],
+		'en_US' => [2, '.', '']
```

Comment:
@manicki As far as I can see there is no custom Sniff in the Mediawiki
standard that checks the space inside brackets, only parentheses. I've
created an issue: https://phabricator.wikimedia.org/T134982

2016-05-11 13:33 GMT+02:00 manicki <notifications@github.com>:

> In src/Presentation/AmountFormatter.php
> <https://github.com/wmde/FundraisingFrontend/pull/389#discussion_r62831576>
> :
>
> > @@ -14,8 +14,8 @@ class AmountFormatter {
> >  	private $locale;
> >
> >  	private $localeData = [
> > -		'de_DE' => [2, ',', '' ],
> > -		'en_US' => [2, '.', '' ]
> > +		'de_DE' => [2, ',', ''],
> > +		'en_US' => [2, '.', '']
>
> You could use Mediawiki custom sniff for it if really wanted to have PHPCS
> complaining. It also has a hidden ability to remove unneeded elements
> from the array <https://phabricator.wikimedia.org/T134857>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wmde/FundraisingFrontend/pull/389/files/65e717ca942b891426f737033ce8b76dbfbf2b66#r62831576>
>



-- 
Gabriel Birke
Software Developer

Wikimedia Deutschland e. V. | Tempelhofer Ufer 23-24 | 10963 Berlin
Phone: +49 (0)30 219 158 26-0
http://wikimedia.de

Imagine a world, in which every single human being can freely share in the
sum of all knowledge. That‘s our commitment.

Wikimedia Deutschland - Gesellschaft zur Förderung Freien Wissens e. V.
Eingetragen im Vereinsregister des Amtsgerichts Berlin-Charlottenburg unter
der Nummer 23855 B. Als gemeinnützig anerkannt durch das Finanzamt für
Körperschaften I Berlin, Steuernummer 27/681/51985.